src/index.ts

```diff
@@ -30,8 +33,10 @@ async function launchReScript(watch: boolean): Promise<ReScriptProcess> {
 
   function dataListener(chunk: any) {
     const output = chunk.toString().trimEnd();
-    // eslint-disable-next-line no-console
-    console.log(logPrefix, output);
+    if (!silent) {
+      // eslint-disable-next-line no-console
+      console.log(logPrefix, output);
+    }
```

Comment:
I'd prefer previous implementation where `stdout` and `stderr` aren't consumed if `silent` is `true`.

I may overlook of benefit consuming, would you please elaborate on that?

Reply:
This overhead is also incurred even if silent is disabled. I thought it would be beneficial to keep the scope of the flag narrow to simplify the implementation, given that there are not any real pain for users now.

Reply:
I see. Thank you for the explanation. I appreciate the simplicity of the implementation, it is really neat 👍